ui.frontend/src/main/webpack/site/js/buildSearch.ts

```diff
@@ -20,7 +20,27 @@ export const buildSearch = async (
     return
   }
 
-  const { callbacks, autoSuggestionDebounceTime = 500 } = options || {}
+  const { trackingUrl } = searchConfig
+
+  const { callbacks = {}, autoSuggestionDebounceTime = 500 } = options || {}
+
+  if (trackingUrl) {
+    callbacks.onSearchItemClick = (url: string) => {
+      const headers = new Headers()
+      headers.append('Content-Type', 'application/x-www-form-urlencoded')
+      fetch(`${window.location.origin}${trackingUrl}`, {
+        method: 'POST',
+        headers: headers,
+        body: `trackedUrl=${url}`,
+      })
+        .then((response) => {
+          console.log(response.status)
+        })
+        .catch((error) => {
+          console.error('Error:', error)
```

Comment:
- [ ] please add more text on the error. E.g. "Error happend while tracking search result item: "

Reply:
**Thought**

Do we capture those console errors somewhere in AEM logs? Eg. errors.log

Reply:
yes in AEM we should capture those as well. left comments: https://github.com/valtech-ch/saas-aem-module/pull/146/files#r817919901